import/import.properties

```diff
@@ -31,3 +31,6 @@ marc.include_errors = true
 # Standards Annex #15, located at <http://unicode.org/reports/tr15/>.
 #
 # marc.unicode_normalize = C
+
+# coordinate error log location - where to put coordinate indexing error logs
+coordinate.log.path = /usr/local/vufind/import
```

Comment:
Could we make this default to solrmarc.path if not set? Then we could leave it commented out by default and reduce the number of hard-coded paths necessary to get SolrMarc working. (But we'd still have the option to send the logs somewhere else when desired). Otherwise, I'm concerned that when people upgrade, they'll start getting errors because of a missing or incorrect setting here.

Reply:
rewriting this so that coordinate.log.path is set in vufind.properties. Default is VUFIND_LOCAL_DIR/import and failing that VUFIND_HOME/import and failing that an error message will be output to indicate that error message will not be logged in a file.